File: src/test/java/helper/jacksonTest/test2/Book.java
package helper.jacksonTest.test2;

public class Book {
    private int bookId;//书的ID
    private String author;//作者
    private String name;//书名
    private int price;//书价

    public int getBookId() {
        return bookId;
    }

    public void setBookId(int bookId) {
        this.bookId = bookId;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    @Override
    public String toString() {
        return "Book [bookId=" + bookId + ", author=" + author + ", name="
                + name + ", price=" + price + "]";
    }

}

File: src/test/java/helper/jacksonTest/test3/Book.kt
package helper.jacksonTest.test3

class Book {
    //书的ID
    var bookId = 0

    //作者
    var author = ""

    //书名
    var name = ""

    //书价
    var price = 0
    override fun toString(): String {
        return "Book(bookId=$bookId, author='$author', name='$name', price=$price)"
    }
}